baseplate/integration/pyramid.py

```diff
@@ -107,13 +127,25 @@ class BaseplateConfigurator(object):
 
     """
 
-    # TODO: remove the default on trust_trace_headers once apps are updated
-    def __init__(self, baseplate, trust_trace_headers=False,
-                 edge_context_factory=None):
+    def __init__(self, baseplate, trust_trace_headers=None,
+                 edge_context_factory=None, header_trust_handler=None):
         self.baseplate = baseplate
-        self.trust_trace_headers = trust_trace_headers
+        self.trust_trace_headers = bool(trust_trace_headers)
+        if trust_trace_headers is not None:
+            warn_deprecated(
+                """setting trust_trace_headers is deprecated in favor of using
```

Comment:
does this end up with a bunch of leading whitespace in the middle of the sentence? you could instead do 

```python
warn_deprecated(
    "blah blah blah "
    "blah blah blah"
)
```

and python will auto-concat those two strings